.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - 0.12
```

Comment:
Could we use the same versions as the generator produces? `0.10`, `0.12`, `4.2`, and `"node"`